starter-code/src/app/components/my-home/my-home.component.html

```diff
@@ -0,0 +1,13 @@
+<p>
+  my-home works!
+</p>
+
+<h1>Movies</h1>
+
+<div>
+  <div *ngFor="let movie of movieArray">
+    <p>{{movie.title}}</p>
+    <p>Room {{movie.room}}</p>
+    <a [routerLink]="['/movie',movie]"><img [src]="[movie.poster]"></a>
```

Comment:
`[routerLink]="['/movie',movie]"` you should have the `movie.id` instead of the whole object.